nntrainer/tensor/blas_interface.cpp

```diff
@@ -847,10 +847,10 @@ unsigned int isamax(const unsigned int N, const float *X, const int incX) {
 #endif
 }
 
-void sine_transformation(const unsigned int N, float *X, float *Y,
+void sine(const unsigned int N, float *X, float *Y,
```

Comment:
Below, sin and cos are used.
However, sine and cosine are used here.
Is it impossible to unify as one?